distributed/scheduler.py

```diff
@@ -503,6 +546,7 @@ def used_resources(self):
     def versions(self):
         return self._versions
 
+    @ccall
     def clean(self):
```

Comment:
Need to wait for Python 3.6 to be dropped before we can annotate this with the `return` type since we need to reference `WorkerState`, which is still being defined. Once that happens we can add `from __future__ import annotations`. That said, this method is used infrequently and knowing the `return` type doesn't look particularly useful in those places. So this is non-pressing. Just noting for future reference.

https://github.com/cython/cython/issues/3935#issuecomment-739115351